### DISCUSSION TITLE
Any API for adjust collision margin in pybullet ?

### DISCUSSION THREAD

**Discussion by v-wewei** [bulletphysics/bullet3#3668] [General]
Since Collision margin play a quite important variable for assemble and grasp task, so would you like to add such a api for us for convinient. @erikogenvik 
Much Thanks for your effort for such a great physic simulator . Hope this suggestion will be accepted.

**Comment by erwincoumans** [comment]
Yes, it can be exposed. See also https://github.com/bulletphysics/bullet3/issues/2584
Will do it soon, but note that you will have to resolve any issues due to changing the default collision margins.

**Comment by erwincoumans** [comment]
See https://github.com/bulletphysics/bullet3/pull/2626

**Comment by v-wewei** [comment]
> See #2626 
Does it support now ? i see the tag is unsupported. @erwincoumans 
seems the c++ version support the option for collision margin, right?

**Comment by v-wewei** [comment]
@erwincoumans 
I am not able to adjust the collision margin for the gripper of my yumi-robot, seems the API can only change the collision margin for basic shape like cylinder and sphere and so on. Would it be able to adjust collision margin for any other convex hull just like the collision in blender. Thanks very much!

**Comment by erwincoumans** [comment]
Can you share a reproduction case with actual urdf and meshes (all in a zip file)?

**Comment by v-wewei** [comment]
@erwincoumans the file is more than 10 MB, how can i show it to you ？

**Comment by v-wewei** [comment]
would you like to provide your email address for convinience , that i can send you through the eamil . the total zip file size is about 40MB large.

在2020-03-02 13:35:33，erwincoumansnotifications@github.com写道：

Can you share a reproduction case with actual urdf and meshes (all in a zip file)?

**Comment by roto5296** [comment]
How did you solve this?
I'm facing a similar problem
